src/start/qemu.md

```diff
@@ -488,11 +515,11 @@ This reset handler will eventually call our main function. Let's skip all the
 way there using a breakpoint and the `continue` command:
 
 ```console
-break main
+break hello::__cortex_m_rt_main
```

Comment:
_Technically_, the name `__cortex_m_rt_main` is an implementation detail of cortex-m-rt and probably shouldn't be relied on for this. I'm not sure if there's any other symbol we could use for a breakpoint, though.

Reply:
If exposing that implementation detail is not desired, I believe this slightly more verbose version would work well for first time users:

```console
list main
```
This will show the source code, from the file examples/hello.rs

```text
6       extern crate panic_halt;
7
8       use cortex_m_rt::entry;
9       use cortex_m_semihosting::{debug, hprintln};
10
11      #[entry]
12      fn main() -> ! {
13          hprintln!("Hello, world!").unwrap();
14
15          // exit QEMU
```
We now would like to add a breakpoint just before the "Hello, world!", which is on line 13. We do that with the `break` command:

```console
break 13
```

Reply:
I updated the PR, replacing setting up a breakpoint at the main symbol with a gentle "set your breakpoint on line 13". I think this way of doing will work well for people following this text.